### PR TITLE
Bug Session seems to expire before 10min/closed#200

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closed #200

### CODE REVIEW

File: lib/load/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function route(app, conf) {
     }
 
     if (secure) {
-      middlewares.splice(0, 0, auth({ permission: prop.permission }));
+      middlewares.splice(0, 0, auth({ skipPermissions: prop.skipPermissions }));
```
Comment (James1x0) [inline lib/load/index.js]: permission still needs to be passed to the auth middleware. Otherwise, this opens up vulnerabilities.

Comment (JavaLavaMT) [reply lib/load/index.js]: to do with are you talking about adding a "Gate" or @parmas? sorry man, been reading on middleware

Comment (James1x0) [reply lib/load/index.js]: @JavaLavaMT The argument to auth is just an object, we need both the `permission` and `skipPermissions` keys.

File: lib/load/index.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ function route(app, conf) {
     }
 
     if (secure) {
-      middlewares.splice(0, 0, auth({ permission: prop.permission }));
+      middlewares.splice(0, 0, auth({ skipPermissions: prop.skipPermissions }, {permissions: prop.permissions}));
```
Comment (James1x0) [inline lib/load/index.js]: This now passes **2** arguments to the `auth` function. The `auth` function expects a single argument that is an object.

**Comment by James1x0** [comment]
@JavaLavaMT description on this PR needs to use the "closes" keyword, not "closed".

**Pull request merged**